### PR TITLE
Document cross-repo pull issue

### PR DESCRIPTION
**Pull request by miminar**
Origin issue [#7729](https://github.com/openshift/origin/issues/7792)

Signed-off-by: Michal Minar miminar@redhat.com

### CODE REVIEW

**Comment by miminar** [comment]
@legionus, @soltysh for technical review.

**Comment by legionus** [comment]
LGTM

File: install_config/install/docker_registry.adoc [hunk 1]
```diff
@@ -969,6 +969,41 @@ link:#session-affinity-workaround[One possible workaround] is to set the
...
+link:../../architecture/core_concepts/builds_and_image_streams.html#adding-tag[re-tagging]
+of a built image into arbitrary image stream like this:
+
+====
+----
+$ oc tag srcimagestream:latest anyproject/pullimagestream:latest
+----
+====
+
+And subsequently pulling from it (using image reference such as
```
Comment (soltysh) [inline install_config/install/docker_registry.adoc]: replace opening parenthesis with a comman

Comment (soltysh) [inline install_config/install/docker_registry.adoc]: put colon after such as

**Comment by soltysh** [comment]
Nits, otherwise LGTM

File: install_config/install/docker_registry.adoc (hunk 1 above)
Comment (miminar) [inline install_config/install/docker_registry.adoc]: Ah, forgot about that one, thanks!

**Comment by miminar** [comment]
Nits fixed.

**Comment by miminar** [comment]
@adellape may I use your merge powers if the words and style please your eyes?

**Pull request merged**